components/Core/Carousel/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC, useState } from 'react'
+import { FlatButton, VerticalLayout, HorizontalLayout } from '../'
+import styles from './Carousel.module.css'
+
+interface CarouselProps {
+  children: React.ReactNode[]
+}
+
+
+const Carousel: FC<CarouselProps> = ({ children }) => {
+  const [ currentIndex, setCurrentIndex ] = useState<number>(0)
+  const prevItem = () => {
+    if (children.length < 2) {
+      return
+    }
+
+    if (currentIndex === 0) {
+      setCurrentIndex(children.length - 1)
+    } else {
+      setCurrentIndex(currentIndex - 1)
+    }
+  }
+  const nextItem = () => {
+    if (children.length < 2) {
+      return
+    }
+
+    const isLastIndex = currentIndex === children.length - 1
+
+    if (isLastIndex) {
+      setCurrentIndex(0)
+    } else {
+      setCurrentIndex(currentIndex + 1)
+    }
+  }
+
+  return (
+    <VerticalLayout>
+      {children[currentIndex]}
+      <HorizontalLayout className={styles.actions}>
+        <FlatButton onClick={prevItem}>{'<'}</FlatButton>
```

Comment:
Using character > or < for arrow is OK, but the rendering might differ if font is changed. I would opt to find or create SVG for this purpouse